src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/RepositoryCloneManager.cs

```diff
@@ -42,7 +43,8 @@ Task<ILocalGitRepo> PrepareCloneAsync(
         SourceMapping mapping,
         IReadOnlyCollection<string> remotes,
         string checkoutRef,
-        CancellationToken cancellationToken);
+        bool resetToRemote = false,
```

Comment:
Can you add a doc for this?

Reply:
added docs for all methods in the interface